src/components/__tests__/ConfirmActionDialog.spec.js

```diff
@@ -21,8 +18,10 @@ jest.mock('@dhis2/ui', () => {
 test('ConfirmActionDialog renders confirm delete dialog', () => {
     const { container } = render(
         <ConfirmActionDialog
-            action={ACTION_DELETE}
-            dashboardName="Twilight Sparkle"
+            title="Delete dashboard"
+            message="Deleting dashboard Twilight Sparkle will remove it for all users. This action cannot be undone. Are you sure you want to permanently delete this dashboard?"
```

Comment:
how can this test pass when the implementation code has quotes around the name?

> message={i18n.t(
                        'Deleting dashboard "{{ dashboardName }}" will remove it for all users. This action cannot be undone. Are you sure you want to permanently delete this dashboard?',
                        { dashboardName: dashboard.name }
                    )}

Reply:
The test isn't testing that i18n string. It is only a test of the ConfirmActionDialog, which just accepts a message property and displays it. (the code you quoted is from ActionsBar)